test/annexB/language/statements/do-while/labeled-fn-stmt.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: It is a Syntax Error if IsLabelledFunction(Statement) is true.
+negative: SyntaxError
+id: sec-labelled-function-declarations
+es6id: B.3.2
+---*/
+
+do label1: label2: function f() {} while (false)
```

Comment:
Isn't `LabelledItem: FunctionDeclaration` disallowed **only** in **strict** mode? I believe this file needs the `onlyStrict` flag because of that.